test/binCases/watch/info-verbosity-off/info-verbosity-off.test.js

```diff
@@ -22,12 +50,39 @@ test("info-verbosity-off", async(done) => {
 		"--info-verbosity",
 		"none"
 	]);
-	const { stdout, stderr } = result;
-	expect(stdout).toEqual(expect.anything());
-	expect(stdout).toContain("");
-	expect(stdout).not.toContain("webpack is watching the files…");
-
-	expect(stderr).toHaveLength(0);
-	expect(stdout).toMatchSnapshot();
-	done();
+
+	var outputCount = 0;
+	var hash1;
+
+	webpackProc.stdout.on("data", data => {
+		data = data.toString();
+
+		if (outputCount === 0) {
```

Comment:
if outputCount is 0 says me nothing, add comment to explain it's the first output count. You should extract this to a function explaining what it does in more detail